src/app/tl-mega-menu/tl-mega-menu-section/tl-mega-menu-section.component.scss

```diff
@@ -19,14 +20,22 @@
   }
   wvr-nav-list-element {
     ::ng-deep {
+      
+      ul{
+        height: auto;
+      }
+
       tl-nav-li {
         width: 100%;
+        
+         
         a {
-          padding: 4px 0px 4px 25px;
+          padding: 4px 0px 6px 25px;
+          margin-bottom:2px;
           color: var(--tl-deep-grey);
           font-weight: lighter;
           width: 100%;
-          display: block;
+          display: block;    
```

Comment:
Please remove the extra spaces at the end of this line.

Reply:
it looks like this is outdated - i went to the files and these had already been corrected